node/runtime/src/lib.rs

```diff
@@ -491,11 +491,19 @@ impl From<CommitteeMember<CrossChainPublic, SessionKeys>> for BlockAuthor {
 		}
 	}
 }
+
+#[cfg(feature = "runtime-benchmarks")]
+pub struct PalletBlockProductionLogBenchmarkHelper;
+
 #[cfg(feature = "runtime-benchmarks")]
-impl From<[u8; 32]> for BlockAuthor {
-	fn from(arr: [u8; 32]) -> Self {
+impl pallet_block_production_log::benchmarking::BenchmarkHelper<BlockAuthor>
+	for PalletBlockProductionLogBenchmarkHelper
+{
+	fn producer_id() -> BlockAuthor {
+		let arr =
+			hex_literal::hex!("9d61b19deffd5a60ba844af492ec2cc44449c5697b326919703bac031cae7f60");
```

Comment:
it is somehow possible in other parts of the code to invject invalid ECDSA key in production part of the code?

Reply:
In CLIs but it is harmless, because invalid key will crash CLI only when someone gives invalid key.

Node doesn't sign with these keys. Node uses keys from keystore.

Reply:
But I'll try to construct public key in a cheaper way. Going from seed (private) to public is extra cost.